api/HTMLIFrameElement.json

```diff
@@ -184,10 +184,10 @@
               "version_added": "25"
             },
             "safari": {
-              "version_added": "10"
+              "version_added": "10.1"
```

Comment:
I wanted to check one of the 10&rarr;10.1 changes and picked this, `allowFullscreen`. The change was in https://trac.webkit.org/changeset/205686/webkit, at WebKit trunk 603.1.6. That points to Safari 10.1 indeed, Safari 10 was branched before this point. Confirmed!